cookbook/security/acl.rst

```diff
@@ -52,20 +52,36 @@ First, you need to configure the connection the ACL system is supposed to use:
 
         # app/config/security.yml
         security:
+            # ...
+
             acl:
                 connection: default
 
     .. code-block:: xml
 
         <!-- app/config/security.xml -->
-        <acl>
-            <connection>default</connection>
-        </acl>
+        <?xml version="1.0" encoding="UTF-8"?>
+        <srv:container xmlns="http://symfony.com/schema/dic/security"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:srv="http://symfony.com/schema/dic/services"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+            <config>
+                <!-- ... -->
+
+                <acl>
+                    <connection>default</connection>
```

Comment:
while this works, we always use attributes for non-array nodes (making this `<acl connection="default"/>`